test/suites/migration.sh

```diff
@@ -11,20 +11,35 @@ test_migration() {
   fi
 
   lxc init testimage nonlive
-  # test moving snapshots
-  lxc snapshot l1:nonlive
+  # FIXME: make this backend agnostic
+  if [ "${LXD_BACKEND}" = "dir" ]; then
+    # Turns out most of this (snapshot transfer) is completely horked on
+    # non-dir backends :)
+    # test moving snapshots
+    lxc snapshot l1:nonlive
+  fi
```

Comment:
Shouldn't hurt to always create the snapshot, just test less stuff if not on dir backend.